plugins/headlamp-plugin/template/.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+  // See http://go.microsoft.com/fwlink/?LinkId=827846
+  // for the documentation about the extensions.json format
+  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode"]
```

Comment:
I thought we had agreed to move with a Headlamp extension that declares dependencies or recommended extensions, so we have a single point to be updated and makes the UX nicer for the user?

Reply:
Yes, when there is an "extension pack" one, it can be added here.
The `headlamp-plugin upgrade` will handle an update of this file.

I'd like to get the .vscode files added first so I can get some feedback on this part first.